### PR TITLE
GH#36916: fixed headings for procedure

### PR DESCRIPTION
**Pull request by JoeAldinger**
4.6+

Github issue #36916

[Preview](https://deploy-preview-42451--osdocs.netlify.app/openshift-enterprise/latest/openshift_images/templates-using-ruby-on-rails.html)

Fixes heading for procedure 
I will need labels and merge, please and thanks.

### CODE REVIEW

File: modules/templates-rails-creating-frontend-service.adoc [hunk 1]
```diff
@@ -48,7 +48,7 @@ env": [
         "value": "postgresql"
     }
 
-],
+]
```
Comment (opayne1) [inline modules/templates-rails-creating-frontend-service.adoc:51]: Only thing I am not sure about is editing this output. You'll need QE to verify. Fixing _Example output_ looks good to me.

**Review by opayne1 — COMMENTED** [review]
Just one comment from me :)

File: modules/templates-rails-creating-frontend-service.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/templates-rails-creating-frontend-service.adoc:51]: +1
If you're changing the contents of a codeblock, you need to get QE to confirm. If you had only changed line 30, we could merge.

Comment (JoeAldinger) [reply modules/templates-rails-creating-frontend-service.adoc]: Sorry about that. I did that by mistake. I changed it back to the original.

**Pull request merged**